src/database/UIDatabase.js

```diff
@@ -50,6 +52,34 @@ const translateToCoreDatabaseType = type => {
   }
 };
 
+const isBooleanData = data => {
```

Comment:
nice! ( I thought when first reviewing that it would just work anyway, but all the same I really like being explicit and using typing )

Reply:
![thankyou](https://user-images.githubusercontent.com/43223637/77487126-6fea5780-6e96-11ea-8caf-3576e18d9242.gif)